ui/src/variables/components/VariableCard.tsx

```diff
@@ -47,35 +47,37 @@ class VariableCard extends PureComponent<Props & WithRouterProps> {
     const {variable, onDeleteVariable} = this.props
 
     return (
-      <ResourceList.Card
+      <ResourceCard
         testID="resource-card"
-        labels={() => this.labels}
-        contextMenu={() => (
+        labels={this.labels}
+        contextMenu={
           <VariableContextMenu
             variable={variable}
             onExport={this.handleExport}
             onRename={this.handleRenameVariable}
             onDelete={onDeleteVariable}
           />
-        )}
-        name={() => (
-          <ResourceList.Name
-            hrefValue={this.editVariablePath}
+        }
+        name={
+          <ResourceCard.Name
+            onClick={this.handleNameClick}
             name={variable.name}
           />
-        )}
-        metaData={() => [<>Type: {variable.arguments.type}</>]}
+        }
+        metaData={[<>Type: {variable.arguments.type}</>]}
       />
     )
   }
 
-  private get editVariablePath(): string {
+  private handleNameClick = (): void => {
     const {
       variable,
       params: {orgID},
     } = this.props
 
-    return `/orgs/${orgID}/variables/${variable.id}/edit`
+    const url = `/orgs/${orgID}/variables/${variable.id}/edit`
+
+    window.location.assign(url)
```

Comment:
I think we should be using router here instead?

Reply:
Good catch!